src/analyze/analyze.c

```diff
@@ -981,7 +981,7 @@ static int list_dependencies_one(sd_bus *bus, const char *name, unsigned level,
                 return r;
 
         STRV_FOREACH(c, deps) {
-                times = hashmap_get(unit_times_hashmap, *c);
+                times = hashmap_get(unit_times_hashmap, *c); /* lgtm [cpp/inconsistent-null-check] */
```

Comment:
is it possible to mark it on the service's database instead, like on coverity?

Reply:
I guess we could just use the "hide from the results" function (in both LGTM and CodeQL UIs).

Reply:
yeah, sounds more palatable than annotating the code directly, given it's parser-specific